core/controllers/reader.py

```diff
@@ -310,18 +312,12 @@ def post(self, exploration_id):
         Args:
             exploration_id: str. The ID of the exploration.
         """
-        exp_issue_dict = self.payload.get('exp_issue_dict')
-        try:
-            unused_exp_issue = stats_domain.ExplorationIssue.from_backend_dict(
-                exp_issue_dict)
-        except utils.ValidationError as e:
-            raise self.InvalidInputException(e)
-
         playthrough_data = self.payload.get('playthrough_data')
         try:
             unused_playthrough = stats_domain.Playthrough.from_backend_dict(
                 playthrough_data)
-        except utils.ValidationError as e:
+            issue_schema_version = self.payload['issue_schema_version']
```

Comment:
nit: move this into its own try/except block to make it obvious it's unrelated to a properly-formatted playthrough.